File: PermCheck/php/TestPermCheck.php
<?php
require 'PermCheck.php';

class Test extends PHPUnit_Framework_TestCase {

	protected $perm_check;

	protected function setup() {
		$this->perm_check = new \Codility\PermCheck;
	}

	public function testSolution() {
		$this->assertEquals(1, $this->perm_check->solution([4,1,3,2]));
	}
}

File: PermCheck/ruby/TestPermCheck.rb
require_relative 'PermCheck'
require 'minitest/autorun'

class Test < MiniTest::Test

	def setup
		@perm_check = PermCheck.new
	end

	def test_solution
		assert_equal 1, @perm_check.solution([4,1,3,2])
	end

end
